configs/_base_/schedules/mmcls/imagenet_bs1024_greedynas.py

```diff
@@ -0,0 +1,6 @@
+# optimizer
+optimizer = dict(type='SGD', lr=0.12, momentum=0.9, weight_decay=4e-5)
```

Comment:
As mentioned in subsection 5.1 of the paper, when training supernet, they use SGD with **Nesterov acceleration**, but that option does not seem to be set here.